package.json

```diff
@@ -101,6 +101,7 @@
     "moment": "^2.24.0",
     "moment-timezone": "^0.5.27",
     "morgan": "^1.9.1",
+    "opossum": "^4.2.0",
```

Comment:
How much does this add to our current JS bundle size?

Reply:
This adds 2.53kb to the bundle size.